tests/versioned/apollo-federation/federated-gateway-server-setup.js

```diff
@@ -44,11 +44,17 @@ function setupFederatedGatewayServerTests(options, agentConfig) {
       // Do after instrumentation to ensure express isn't loaded too soon.
       const apollo = require('apollo-server')
 
-      // Sub-graph services are currently auto-instrumented via express.
-      // Ignore transaction plugin will prevent creation of standard data and indicate
-      // to tests we do not intend to assert on these transactions.
-      const ignoreTransactionPlugin = createIgnoreTransactionPlugin(nrApi)
-      const subGraphPlugins = [ ignoreTransactionPlugin ]
+      let subGraphPlugins = []
+
+      if (options.instrumentSubGraphs) {
```

Comment:
Seems like this is a new option. We should document when with this issue: https://github.com/newrelic/newrelic-node-apollo-server-plugin/issues/88

Reply:
this is just in the testing code not the plugin

Reply:
Derp, vacation brain still loading ... 😄